Resources/Textures/Shaders/crispy.swsl

```diff
@@ -0,0 +1,351 @@
+//
+// Description : Array and textureless GLSL 2D simplex noise function.
```

Comment:
did you confirm the shader works in compatibility mode?

Reply:
Confirmed to be working on my machine.